configuration.php

```diff
@@ -196,7 +196,9 @@ function BuildFileList($returnjson=false){
 				$List=explode(", ",$_REQUEST[$key]);
 				$config->ParameterArray[$key]=$List;
 			}else{
-				$config->ParameterArray[$key]=$_REQUEST[$key];
+				if (isset($_REQUEST[$key])) {

```

Comment:
Should probably check that $_REQUEST[$key] is defined, otherwise we end up overwriting config in the database which wasn't intended.